eth-connector/src/deposit_event.rs

```diff
@@ -260,6 +259,7 @@ impl DepositedEvent {
                 kind: ParamType::Uint(256),
                 indexed: false,
             },
+            // NOTICE: fee field is currently unused
```

Comment:
But should we have this event parameter anyway? Is it some kind of standard?

Reply:
It was wrong comment. Deleted